src/index.html

```diff
@@ -9,10 +9,44 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The `href` attribute in the `<link>` tag should point to a CSS file, not a SCSS file. SCSS is a preprocessor language and needs to be compiled to CSS before being used in HTML.